src/components/CloseAppButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { IoCloseCircleOutline } from 'react-icons/io5'
+
+export const CloseAppButton = () => {
+  return (
+    <Link to="/">
+      <CloseIcon />
+    </Link>
+  )
+}
```

Comment:
same https://github.com/aberllin/react-apps-dashboard/pull/3/commits/acc85feeb602be4aad0f8aba2a567be8656222b8#r552405695

Reply:
This component is used only in AppWindow, so you can create a folder components/AppWindow and move this component there